### ISSUE TITLE
function test succeeds when function run fails

### ISSUE THREAD

**Issue by AdrienLemaire** [arabold/serverless-test-plugin#4]
Using serverless-test-plugin@0.2.0, it looks like all tests return success whether they succeed or fail:

```
$ sls function run test-token
Serverless: Running test-token...  
Serverless: -----------------  
Serverless: Failed - This Error Was Returned:  
Serverless: undefined  
Serverless: undefined 
$  sls function test test-token
Serverless: Testing test-token...  
Serverless: Success!  
Serverless: -----------------  
```

**Comment by arabold** [comment]
Yuck! I'll take a look.

**Comment by AdrienLemaire** [comment]
@arabold have you got a chance to look into it? Could you reproduce it or is it only my issue?

**Comment by arabold** [comment]
@Fandekasp , I've released a new version 0.2.1 that should fix this. Sorry for the trouble.
